app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Kindly change the `:user` to `:author` and remove the `foregin_key`. It's only necessary to set up the association between the `Post` model and the `User` model to specify the changed name to the model which is the `User` and tell the `Post` model that this is actually the `User` `class_name`.

> Please apply the changes in all similar situations through the associations

Reply:
Thank you for the feedback.